config/locales/en.yml

```
@@ -171,7 +171,7 @@ en:
       direct_likes: "Likes on my posts"
       indirect_comment: "Comments on all posts I've commented on"
       moderation_email: "Moderation emails"
-      email_notifications: "Email notifications are sent for each post in topics you follow. If you prefer to receive a digest, choose one of these options:"
+      email_notifications: "email_notifications are sent for each post in topics you follow. If you prefer to receive a digest, choose one of these options:"
```

Ah, here I think the issue is that we already had a translation for the text "email_notifications" so perhaps we should make the new one "email_notifications_title" to distinguish it. I'll make a suggestion which you should be able to accept. Also, we don't need to remove "Email notifications" from the start of this line as we can leave the existing translation as it was. 
```suggestion
      email_notifications: "Email notifications are sent for each post in topics you follow. If you prefer to receive a digest, choose one of these options:"
      email_notifications_title: "Email Notifications"
```